### ISSUE TITLE
Support TypeScript 4.7 "moduleResolution": "node16"

### ISSUE THREAD

**Issue by pschiffmann** [fullcalendar/temporal#17]
TypeScript can't find the type definitions for this package if I set the [moduleResolution](https://www.typescriptlang.org/tsconfig#moduleResolution) option to `node16` in my `tsconfig.json`.

Adding a `"types"` key to the package.json exports section fixes the problem:

```json
{
  "exports": {
    ".": {
      "types": "./index.d.ts",
      "require": "./index.cjs",
      "import": "./index.mjs"
    }
  }
}
```

This needs to be patched in both `temporal-polyfill` and `temporal-spec` packages.

I can send PRs if you want.

**Comment by arshaw** [comment]
Released in v0.1.0

**Comment by arshaw** [comment]
I realized this needed fixing in the `temporal-spec` package as well. Fixed here: https://github.com/fullcalendar/temporal/commit/e955c2b460d7fdc0321b283cb5d36b7700c509b3

Released in v0.1.1
